hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/management/HotRestartBackupInterruptMessageTask.java

```diff
@@ -0,0 +1,91 @@
+package com.hazelcast.client.impl.protocol.task.management;
+
+import com.hazelcast.client.impl.protocol.ClientMessage;
+import com.hazelcast.client.impl.protocol.codec.MCInterruptHotRestartBackupCodec;
+import com.hazelcast.client.impl.protocol.task.AbstractInvocationMessageTask;
+import com.hazelcast.instance.impl.Node;
+import com.hazelcast.internal.management.ManagementCenterService;
+import com.hazelcast.internal.management.operation.AbstractManagementOperation;
+import com.hazelcast.internal.nio.Connection;
+import com.hazelcast.spi.impl.operationservice.InvocationBuilder;
+import com.hazelcast.spi.impl.operationservice.Operation;
+
+import java.security.Permission;
+
+import static com.hazelcast.internal.management.ManagementDataSerializerHook.HOT_RESTART_BACKUP_INTERRUPT_OPERATION;
+
+class HotRestartBackupInterruptOperation extends AbstractManagementOperation {
+
+    private final Node node;
+
+    public HotRestartBackupInterruptOperation(Node node) {
+        this.node = node;
+    }
+
+    @Override
+    public void run()
+            throws Exception {
+        node.getNodeExtension().getHotRestartService().interruptBackupTask();
```

Comment:
Same with trigger hot backup. This method already causes interruption on all members. It's enough to execute it on a single coordinator member.

Reply:
Fixed, thanks.